File: public/code_samples/dart/signUp.dart
var headers = {
  'Content-Type': 'application/json'
};
var request = http.Request('POST', Uri.parse('http://localhost:4000/api/v1/auth/sign-up'));
request.body = json.encode({
  "username": "johndoe",
  "email": "johndoe@example.com",
  "password": "qwerty"
});
request.headers.addAll(headers);

http.StreamedResponse response = await request.send();

if (response.statusCode == 200) {
  print(await response.stream.bytesToString());
}
else {
  print(response.reasonPhrase);
}

File: public/code_samples/javascript/signUp.js
const myHeaders = new Headers()
myHeaders.append('Content-Type', 'application/json')

const raw = JSON.stringify({
  username: 'johndoe',
  email: 'johndoe@example.com',
  password: 'qwerty'
})

const requestOptions = {
  method: 'POST',
  headers: myHeaders,
  body: raw,
  redirect: 'follow'
}

fetch('http://localhost:4000/api/v1/auth/sign-up', requestOptions)
  .then(response => response.json())
  .catch(error => console.error('error', error))
